src/components/EarningsTable/EarningsTableMainFilter/EarningsTableMainFilter.js

```diff
@@ -59,6 +69,12 @@ function EarningsTableMainFilter(props) {
     setIsMainFilterOpen(false);
   };
 
+  useEffect(() => {
+    api.getOrganizations().then((res) => {
+      setOrganisations(res);
+    });
+  }, [mainFilter]);
+
```

Comment:
More of a question...what happens when the API call fails or returns an error? Should we catch that and log it?

Reply:
Noted. I have added logic to catch error